mmv1/third_party/terraform/services/storage/resource_storage_bucket.go.tmpl

```diff
@@ -549,6 +549,23 @@ func ResourceStorageBucket() *schema.Resource {
 					},
 				},
 			},
+			"hierarchical_namespace": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Description: `The bucket's HNS support, which defines bucket can organize folders in logical file system structure`,
+				Elem : &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enabled": {
+							Type:        schema.TypeBool,
+							Default:     false,
+							Optional:    true,
```

Comment:
Can remove this in favor of `Required: true,`

Reply:
removed Optional and added required true